tests/testprogs/usdt_test.c

```diff
@@ -0,0 +1,21 @@
+#include <sys/sdt.h>
```

Comment:
```
/home/mmarchini/workspace/iovisor/bpftrace/tests/testprogs/usdt_test.c:1:10: fatal error: sys/sdt.h: No such file or directory
 #include <sys/sdt.h>
```

Should we make `systemtap-sdt-dev` a hard dependency for bpftrace? I don't mind, especially if we decide to instrument bpftrace with it in the future.

Reply:
I think so, for the USDT tests it is pretty fundamental. It seems like it's already a development dependency of BCC, as their tests use it.